static/styles/drafts.css

```diff
@@ -90,7 +90,7 @@
     line-height: 1;
     padding-top: 10px;
     padding-bottom: 10px;
-    padding-left: 0px;
+    margin-left: 0px;
```

Comment:
Do we need some space? I feel like it looks a little bit weird lined up right on the left side. Maybe 5px?